tests/Foundation/FoundationAliasLoaderTest.php

```diff
@@ -20,4 +20,10 @@ public function testGetInstanceCreatesOneInstance()
         $loader = AliasLoader::getInstance(['foo' => 'bar']);
         $this->assertSame($loader, AliasLoader::getInstance());
     }
+
+    // unregister dummy not existing classes
```

Comment:
here the comment reason why we need tearDownAfterClass at all not about the implementation of it.